packages/core/strapi/lib/commands/actions/ts/generate-types/command.js

```diff
@@ -10,12 +10,7 @@ module.exports = ({ command }) => {
   command
     .command('ts:generate-types')
     .description(`Generate TypeScript typings for your schemas`)
-    .option(
-      '-o, --out-dir <outDir>',
-      'Specify a relative directory in which the schemas definitions will be generated'
-    )
-    .option('-f, --file <file>', 'Specify a filename to store the schemas definitions')
```

Comment:
Same comment as -o. Might want to allow it with a warning.

Reply:
It won't be possible for this one, we'll make reference to the file from other generated files and we want to have the correct path by default :confused:

Reply:
Also, we'll have multiple files emitted instead of just one

Reply:
Ahh, ok, it makes sense to remove this one then